tests/UI/pages/API/APIbasePage.ts

```diff
@@ -0,0 +1,25 @@
+// Import pages
+import CommonPage from '@pages/commonPage';
+
+import type {Page, Response} from 'playwright';
+import {use, request} from 'chai';
+import chaiHttp from 'chai-http';
+import {SuperAgentRequest} from 'superagent';
+
+use(chaiHttp);
+
+/**
+ * API parent page, contains functions that can be used on all API endpoints
+ * @class
+ * @extends CommonPage
+ */
+class APIBasePage extends CommonPage {
```

Comment:
As it's not a page, we can avoid call it a page and not extend CommonPage.
we can also add a new folder `helpers/api` and move this file.
And I'm not sure a class is the right approach to this kind tests.